src/components/FooterPreLogin/FooterPreLogin.tsx

```diff
@@ -36,7 +37,18 @@ export const FooterPreLogin = ({
         onExit(href, linkType);
       }
     };
-  const { aboutUs, productsAndServices, resources, followUs } = links;
+  const { aboutUs, resources, followUs } = links;
+
+  const [jsonProducts, setJsonProducts] = useState([]);
+
+  useEffect(() => {
+    fetch(url)
+      .then((r) => r.json())
+      .then((json) => setJsonProducts(json))
+      .catch((err) => {
+        throw new Error(err);
```

Comment:
don't throw the error, or you will break the application!
this error will brought just to an invalid configuration of the products to show, it's not a critical error!
let's just notify this event and let the array to be empty

we could add an optional Prop onProductsJsonFetchError, which as default will console.error the error
in selfcare, as an example, it will notify the error to mixPanel through the trackAppError method of analyticsService